src/main/java/com/targomo/client/api/TravelOptions.java

```diff
@@ -1290,7 +1295,7 @@ public Integer getQuadrantSegments() {
 
     /**
      * Set how much the polygons will be simplified (in meters). This can reduce the points in the polygon significantly. todo
```

Comment:
I think this whole comment block is wrong, perhaps simply remove it

Reply:
Removed also from getter